geonode/templates/base.html

```diff
@@ -151,6 +144,28 @@
       <p>Please use a <strong>modern browser</strong> like Mozilla Firefox, Google Chrome or Safari.</p>
     </div>
```

Comment:
This is also an alert for outdated browser. So, it does exists in original code. We have to make sure there is no conflict of interests. There might be existing methods for Geonode to check outdated browser.